lib/evm/eei.ts

```diff
@@ -33,7 +33,6 @@ export interface Env {
 export interface RunResult {
   logs: any // TODO: define type for Log (each log: [Buffer(address), [Buffer(topic0), ...]])
   returnValue?: Buffer
-  gasRefund: BN
```

Comment:
Wouldn't this introduce a breaking change? I think we should duplicate the `gasRefund` value for now, and add a comment about this field being deprecated and removed in the future.

Is this possible @s1na? I'm not completely sure atm.